src/core/dbus-cgroup.c

```diff
@@ -1441,6 +1468,100 @@ int bus_cgroup_set_property(
                         }
                 }
 
+                return 1;
+        } else if (STR_IN_SET(name, "IPFilterIngress", "IPFilterEgress")) {
+                const char *path;
+                char **t;
+                char **entry;
+                char ***filters;
+                size_t n = 0;
+                bool skip_entry;
+
+                filters = streq(name, "IPFilterIngress") ? &c->ip_filters_ingress : &c->ip_filters_egress;
+                r = sd_bus_message_enter_container(message, 'a', "s");
+                if (r < 0)
+                        return r;
+
+                for (;;) {
+                        r = sd_bus_message_read(message, "s", &path);
+                        if (r < 0)
+                                return r;
+                        if (r == 0)
+                                break;
+
+                        if (!path_is_normalized(path) || !path_is_absolute(path))
+                                return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "%s= expects a normalized absolute path.", name);
+
+                        if (!UNIT_WRITE_FLAGS_NOOP(flags)) {
+                                skip_entry = false;
+
+                                STRV_FOREACH(t, *filters) {
+                                        if (path_equal(path, *t)) {
+                                                skip_entry = true;
+                                                break;
+                                        }
+                                }
```

Comment:
please add path_strv_contains() as common helpe to path-util.c (see other comment)